client/js/components/sort-search.js

```diff
@@ -0,0 +1,22 @@
+import showHide from './show-hide';
+import { nodeList } from '../util';
+
+const sortSearch = (el) => {
+  const toggle = showHide({
+    el,
+    activeClass: 'is-sort-search-active'
+  });
+
+  const form = el.querySelector('.js-sort-search-form');
+  const inputEls = nodeList(form.getElementsByTagName('input'));
+
+  inputEls.forEach((el) => {
+    el.addEventListener('change', () => {
+      form.submit();
+    });
+  });
+
+  toggle.trigger.addEventListener('click', toggle.toggleActive);
```

Comment:
it's from the `show-hide.js` component we have that toggles opening/closing of a component section while handling all the required ARIA attrs etc.